lib/scrabble_class.rb

```diff
@@ -0,0 +1,67 @@
+require 'awesome_print'
+
+module Scrabble
+
+  class Scrabble
+    SCORE = {
+      "a" => 1, "e" => 1, "i" => 1, "o" => 1, "u" => 1, "l" => 1, "n" => 1, "r" => 1, "s" => 1, "t" => 1, "d" => 2, "g" => 2, "b" => 3, "c" => 3, "m" => 3, "p" => 3, "f" => 4, "h" => 4, "v" => 4, "w" => 4, "y" => 4, "k" => 5, "j" => 8, "x" => 8, "q" => 10, "z" => 10
+    }
+
+    #this is where point_conversion happens
+    def self.score(word)
+      #to control for case insensitivity
+      #to split into individual strings
+      word = word.downcase.split(//)
+      total_points = 0
+      word.each do |input|
+        points = SCORE[input]
+        total_points += points
+      end
```

Comment:
Excellent use of `.each` and an accumulator (`total_points`). There's another way to do this using a different Enumerable method--check out `.reduce` (sometimes called `.inject`): http://ruby-doc.org/core-2.2.0/Enumerable.html#method-i-reduce